### PR TITLE
Docs limitation and params fix + 5 func

### PR DESCRIPTION
**Pull request by 1e-to**

### CODE REVIEW

**Comment by AlexanderKalistratov** [comment]
@akharche @densmirn @kozlov-alexey ping

**Comment by AlexanderKalistratov** [comment]
@akharche @densmirn @kozlov-alexey sorry, you are not reviewers :)

**Comment by AlexanderKalistratov** [comment]
@1e-to please add reviewer

File: examples/series/series_append.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+"""
+Expect Series
+0      one
+1      two
+2    three
+0     four
+1     five
+2      six
+dtype: object
+"""
```
Comment (densmirn) [inline examples/series/series_append.py]: I would move the example result before import block and replace `:lines: 27-` with ~`:lines: 37-` to not duplicate the result in the documentation. Please note it's not a single place where you can make such change.

**Pull request merged**